.travis.yml

```diff
@@ -99,3 +108,49 @@ jobs:
         - ./mongodb-linux-x86_64-3.6.6/bin/mongod --fork --dbpath ./data/db/27017 --syslog --port 27017
       env:
         - DB_STRING='--dbclient=mongo --dbhost=localhost --dbport=27017 --dbname=strapi_test --dbusername= --dbpassword='
+
+    - <<: *e2e_tests_ee
```

Comment:
What's the difference between `e2e_tests` and `e2e_tests_ee`? Is it possible to use `e2e_tests` twice?

Reply:
the diff is that STRAPI_DISABLE_EE

Reply:
cleaned up a bit